plugins/snapshot_api_objects.yaml

```diff
@@ -0,0 +1,43 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  # 'name' must match the filename of the manifest. The name defines how
+  name: snapshot_api_objects
```

Comment:
Is there a reason you picked _ over - ? It's rather not typical plugin naming.

Reply:
This was an oversight, I've updated it to `snapshot-api-objects` now